content/ja/docs/concepts/workloads/controllers/job.md

```diff
@@ -479,20 +566,15 @@ Events:
 
 ### 可変スケジューリング命令  {#mutable-scheduling-directives}
 
-{{< feature-state for_k8s_version="v1.23" state="beta" >}}
-
-{{< note >}}
-この機能を使うためには、[APIサーバー](/docs/reference/command-line-tools-reference/kube-apiserver/)上で`JobMutableNodeSchedulingDirectives`[フィーチャーゲート](/ja/docs/reference/command-line-tools-reference/feature-gates/)を有効にする必要があります。
-デフォルトで有効になっています。
-{{< /note >}}
+{{< feature-state for_k8s_version="v1.27" state="stable" >}}
 
 ほとんどの場合、並列Jobは、すべてのPodが同じゾーン、またはすべてのGPUモデルxかyのいずれかであるが、両方の混在ではない、などの制約付きで実行することが望ましいです。
 
 [suspend](#suspending-a-job)フィールドは、これらの機能を実現するための第一歩です。Suspendは、カスタムキューコントローラーがJobをいつ開始すべきかを決定することができます。しかし、Jobの一時停止が解除されると、カスタムキューコントローラーは、Job内のPodの実際の配置場所には影響を与えません。
 
-この機能により、Jobが開始される前にスケジューリング命令を更新でき、カスタムキューコントローラーがPodの配置に影響を与えることができると同時に、実際のPodとノードの割り当てをkube-schedulerにオフロードすることができます。これは一時停止されたJobの中で、一度も一時停止解除されたことのないJobに対してのみ許可されます。
+この機能により、Jobが開始される前にスケジューリング命令を更新でき、カスタムキューコントローラーがPodの配置に影響を与えることができるようになります。これは一時停止されたJobの中で、一度も一時停止解除されたことのないJobに対してのみ許可されます。
```

Comment:
In `en` content, the sentence following is included:
- "while at the same time offloading actual pod-to-node assignment to kube-scheduler."
Could you add the translation about this?

https://github.com/kubernetes/website/blob/main/content/en/docs/concepts/workloads/controllers/job.md?plain=1#L751-L752

Reply:
Thank you @atoato88 !
I fixed it.